### DISCUSSION TITLE
How to initialize a LibriTTS Dataset with the Python Trainer API?

### DISCUSSION THREAD

**Discussion by stanleyshly** [coqui-ai/TTS#1379] [General Q&A]
When setting up LibriTTS with `dataset_config = BaseDatasetConfig(name="libri_tts", path=os.path.join(output_path, "../LibriTTS/train-clean-100/"))`, I get this error: 

```
Traceback (most recent call last):
  File "train_fast_speech.py", line 78, in <module>
    train_samples, eval_samples = load_tts_samples(dataset_config, eval_split=True)
  File "/home/workstation/.local/lib/python3.8/site-packages/TTS/tts/datasets/__init__.py", line 111, in load_tts_samples
    meta_data_train = formatter(root_path, meta_file_train, ignored_speakers=ignored_speakers)
  File "/home/workstation/.local/lib/python3.8/site-packages/TTS/tts/datasets/formatters.py", line 249, in libri_tts
    assert os.path.exists(item[1]), f" [!] wav files don't exist - {item[1]}"
KeyError: 1
```
I placed the LibriTTS folder on branch up of the trainer script(exit one folder, and placed the LibriTTS dataset folder there), so I am not sure why it is not working. Any tips for how to properly intialize the LibriTTS in the Python Trainer API?

**Comment by erogol** [comment]
It is actually a bug. item is a dictionary not a list anymore.

**Reply by stanleyshly** [reply]
Issue filed: #1380
